accounts/forms.py

```diff
@@ -19,13 +19,20 @@ class CustomUserCreationForm(UserCreationForm):
     receive_newsletter = forms.BooleanField(
```

Comment:
We could even remove this from the form actually? Unless we plan on doing a newsletter. 🤔